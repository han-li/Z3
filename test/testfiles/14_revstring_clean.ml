let rev_string str =
  let len = String.length str in
  let res = String.create len in
  let last = len - 1 in
  for i = 0 to last do
    let j = last - i in
    res.[i] <- str.[j];
  done;
  (res);;

print_string (rev_string ("Hello my dear how are we today"));;
print_newline ();;
