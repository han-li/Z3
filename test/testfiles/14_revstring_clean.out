yadot ew era woh raed ym olleH
